platform/web/doc_classes/EditorExportPlatformWeb.xml

```diff
@@ -35,6 +35,10 @@
 		</member>
 		<member name="progressive_web_app/enabled" type="bool" setter="" getter="">
 		</member>
+		<member name="progressive_web_app/ensure_cross_origin_isolation_headers" type="bool" setter="" getter="">
```

Comment:
What is the typo?

Reply:
I don't think the lack of a hyphen would be a typo. It still conveys the right meaning. For example "open source" and "open-source" are two widespread spellings for the concept, there's no loss of meaning IMO.

Reply:
Is there an English translation? Or we're stuck with the automatic version?

Reply:
Could add an editor name processor entry to improve the formatting

Reply:
I checked but EditorPropertyNameProcessor splits by `_`, so it can only convert single words.

I don't think it's that big a deal myself, but if it's Absolutely Unacceptable™ not to have a hyphen, I guess we can make an exception for the property name. It's just weird to have a property name that's not a valid identifier.